pysat/_instrument.py

```diff
@@ -58,19 +58,17 @@ class Instrument(object):
         (default=False)
     strict_time_flag : boolean
         If true, pysat will check data to ensure times are unique and
-        monotonically increasing. (default=True)
-    multi_file_day : boolean or NoneType
+        monotonically increasing. (default=True
+    directory_format : string or NoneType
+        Directory naming structure in string format. Variables such as platform,
+        name, and tag will be filled in as needed using python string
+        formatting. The default directory structure would be expressed as
+        '{platform}/{name}/{tag}'. If None, the default directory structure is
+        used. (default=None)
+    multi_file_day : boolean
         Set to True if Instrument data files for a day are spread across
         multiple files and data for day n could be found in a file
-        with a timestamp of day n-1 or n+1.  (default=None)
-    manual_org : bool
-        if True, then pysat will look directly in pysat data directory
-        for data files and will not use default /platform/name/tag
-    directory_format : str
-        directory naming structure in string format. Variables such as
-        platform, name, and tag will be filled in as needed using python
-        string formatting. The default directory structure would be
-        expressed as '{platform}/{name}/{tag}'
+        with a timestamp of day n-1 or n+1.  (default=False)
```

Comment:
If we aren't changing this here, it needs to default to `None` or it will break the Instruments that use it.